clio-dataaccess-model/src/test/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/ElasticsearchIndexSpec.scala

```diff
@@ -64,7 +68,10 @@ class ElasticsearchIndexSpec extends FlatSpec with Matchers with ModelAutoDeriva
     it should "fields for v2 document" in {
       // Snake-case-ify the bookkeeping fields.
       val bookkeeping =
-        Seq(ClioDocument.UpsertIdFieldName, ClioDocument.EntityIdFieldName).map { name =>
+        Seq(
+          ElasticsearchIndex.UpsertIdElasticsearchName,
+          ElasticsearchIndex.EntityIdElasticsearchName
+        ).map { name =>
           keywordField(ElasticsearchUtil.toElasticsearchName(name))
```

Comment:
Shouldn't need the `toElasticsearchName` anymore.

Reply:
Also, since we seem to use the `Seq(UpsertIdName, EntityIdName)` in a bunch of places, it might be worth making another `val BookkeepingNames` to store it, for other code to use.